src/types.ts

```diff
@@ -1,3 +1,36 @@
+import { render } from "@testing-library/react";
+import { mount } from "enzyme";
+
+export type FullProps<C extends React.ComponentType> = React.ComponentProps<C>;
```

Comment:
Is the intent of this `FullProps` just to alias `React.ComponentProps`? If so: would you be up for removing it? `React.ComponentProps` is pretty standard.

Reply:
...then again, looking at my request starting code, maybe the abbreviation is good 😂

Reply:
I'm happy to go either way. I found the (extremely minor) ease of reading when I saw the `FullProps` to be worth it, but I absolutely understand the stance of trimming it out. I'll run through the other changes first and then we can see how it looks 👍

Reply:
I absolutely understand this, I'm wondering if we should rename this something like `FullComponentProps` to remind the reader that we're still talking about components here